src/backend/modules/profile/dto/getUserBuyOffer.query.ts

```diff
@@ -15,5 +16,5 @@ export class GetUserBuyOfferQuery extends Pagination {
   @Type(() => Number)

```

Comment:
I think this line needs to be removed.
We changed from integer to string (enum) so there's no need to transform string anymore.

Also next line "@IsInt()", change it to "@IsString()".